GitUI/UserControls/RevisionGrid/GraphDrawer/SegmentPoints.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace GitUI.UserControls.RevisionGrid.GraphDrawer
+{
+    internal struct SegmentPoints
```

Comment:
Ditto here. Maybe?
```suggestion
    internal struct SegmentPointsInfo
```

Reply:
Thank you very much for the guidance on the names and the `struct` variants, Igor!

Just `SegmentPointsInfo` remains not `readonly` because the `x` coordinates are changed in a loop.